### PR TITLE
Add ignore_if_doc_commented option

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @gkoz

### CODE REVIEW

File: src/regenerate.rs [hunk 1]
```diff
@@ -120,7 +122,7 @@ fn get_corresponding_type(elements: &[(Option<TypeStruct>, Vec<String>)],
 // The hashmap key is `Some(file name)` or `None` for entries that ignore file name
 pub fn regenerate_comments(work_dir: &Path, path: &str,
         infos: &mut HashMap<Option<String>, Vec<(Option<TypeStruct>, Vec<String>)>>,
-        ignore_macros: bool) {
+        ignore_macros: bool, ignore_doc_commented: bool) {
```
Comment (gkoz) [inline src/regenerate.rs]: These are getting unmanageable. Maybe use bitflags to make a single options type?

Comment (GuillaumeGomez) [inline src/regenerate.rs]: The code remains readable for the moment. But yes, if we add another option, I'll use a bitflag instead.

Comment (gkoz) [inline src/regenerate.rs]: Just look at the tests: `false, false` or at [this](https://github.com/gtk-rs/lgpl-docs/blob/master/src/lib.rs#L26). I could add comments there of course but we could easily have something more structured as well.

Comment (GuillaumeGomez) [inline src/regenerate.rs]: Then it'll be the object of another PR. A bigger clean is needed for the code anyway.
